ykbuild/wrap-clang++.sh

```diff
@@ -6,8 +6,5 @@
 
 set -e
 
-export PATH=${DEP_YKBUILD_YKLLVM}:${PATH}
-
-echo "clang++ $@" > $(mktemp -p ${YK_CC_TEMPDIR})
-
-clang++ $@
+echo "${YK_COMPILER_PATH} $@" > $(mktemp -p ${YK_COMPILER_TEMPDIR})
```

Comment:
Isn't this going to grab the `clang` binary instead of the `clang++` one?

Reply:
I'm about 95% certain that we don't need *two* wrappers: we just need one and it can call `clang` or `clang++` depending on what `YK_COMPILER_PATH` is set to. Warning: I might be wrong. But I think we can tackle this in a follow-up clean-up PR.

Reply:
I think you're right actually. However, I think it's calling the wrong binary from ykllvmwrap (https://github.com/ykjit/yk/pull/681#discussion_r1191575427)

Reply:
Ah, I see you replied. Ignore me.

Reply:
Perhaps this script should be named `wrap_compiler.sh`?

Reply:
Yes -- this is one of several clean-up opportunities I've identified. But I think that's for subsequent PR(s).

Reply:
and delete `wrap-clang.sh`?

Reply:
ok